.github/workflows/avm.res.fabric.capacity.yml

```diff
@@ -0,0 +1,88 @@
+name: "avm.res.fabric.capacity"
```

Comment:
Hi, I guess this file doesn't belong here, please remove it from PR.

Reply:
Fixed.